infra/terraform/modules/tezos_node/user_data_rolling.tpl

```diff
@@ -68,16 +68,21 @@ rm snapshot.rolling
 cat > health-logs-analysis.sh << 'EOF'
 #!/bin/bash
 
-if [[ $(docker logs ${network}_node_1 2>&1 | grep 'validator.peer: Fetch of operations ' | grep ' timed out' | wc -l) > 0 ]]; then
-  echo "$(date -R) - health-logs-analysis - Warning : dysfunctionment detected. Killing the node"
+if [[ $(curl localhost:8000/chains/main/blocks/head -w %%{time_total} -o /dev/null --silent) > 1 ]]; then
```

Comment:
I thought we were using the `grep 'validator.peer: Fetch of operations '` method, no ? That is what we used for the last stress test no ?

Reply:
Nop I wanna test a last time with it and it has worked.
I think the problem was about the load which was too important for a too long sync period.